Examples/CaseStudies/SwiftUICaseStudies/02-Effects-WebSocket.swift

```diff
@@ -75,8 +74,7 @@ let webSocketReducer = Reducer<WebSocketState, WebSocketAction, WebSocketEnviron
         WebSocketId(), URL(string: "wss://echo.websocket.events")!, []
       )
       .receive(on: environment.mainQueue)
-      .map(WebSocketAction.webSocket)
-      .eraseToEffect()
+      .eraseToEffect(WebSocketAction.webSocket)
       .cancellable(id: WebSocketId())
```

Comment:
Unrelated: Should all the examples (where possible) use the type based cancellation now?

Reply:
Ideally! I think this example just requires a bit more refactoring to make it work.

Reply:
Cool. It can go in the weekend pile.